azavea/climate-change-components
# WIP: angular update

## Overview: WIP DO NOT MERGE

Upgrade Angular and `rxjs` major versions.

Companion to azavea/temperate#1227.

This has not been tested yet in a parent project, and so is a WIP PR until a companion PR is ready.


### Notes

Based on [this branch](https://github.com/azavea/climate-change-components/tree/feature/mvm/angular-update), with further changes to fix building, type warnings, and tests.


## Testing Instructions

 * `rm -rf node_modules/ && yarn install`
 * `yarn test` should pass
 * `yarn build` should succeed and populate `dist/`
 * `yarn pack` should produce a component package tarball
